### ISSUE TITLE
Marshal NotifyPropertyChangedEventArgs to WinRT throws NullRef

### ISSUE THREAD

**Issue by devsko** [dotnet/corefx#36403]
Another WinRT marshalling. This throws. Complete repro https://github.com/devsko/Bug35909

``` c#
using System;
using System.ComponentModel;
using Windows.UI.Xaml;
using Windows.UI.Xaml.Controls;
using Windows.UI.Xaml.Data;
using Windows.UI.Xaml.Hosting;

namespace ConsoleApp2
{
    class Program
    {
        private class Npc : INotifyPropertyChanged
        {
            public event PropertyChangedEventHandler PropertyChanged;

            private string _value;
            public string Value
            {
                get => _value;
                set
                {
                    _value = value;
                    PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(nameof(Value)));
                }
            }

        }
        static void Main(string[] args)
        {
            using (WindowsXamlManager.InitializeForCurrentThread())
            {
                var npc = new Npc();
                var textBlock = new TextBlock();
                textBlock.SetBinding(TextBlock.TextProperty, new Binding() { Mode = BindingMode.OneWay, Path = new PropertyPath(nameof(Npc.Value)), Source = npc });

                npc.Value = "TEST";
            }
        }
    }
}
```
```
Object reference not set to an instance of an object.

   at System.Runtime.InteropServices.WindowsRuntime.PropertyChangedEventArgsMarshaler.ConvertToNative(PropertyChangedEventArgs managedArgs)
   at ConsoleApp2.Program.NPC.set_Value(String value) in ConsoleApp8\Program.cs:line 24
   at ConsoleApp2.Program.Main(String[] args) in ConsoleApp8\Program.cs:line 37
```
Environment
- Windows Preview 10.0.18351.0
- Windows SDK 10.0.18342.0
- .NET Core 3.0.100-preview4-010965 (local build with PR 36224 included - it gets not thru to core-sdk)

**Comment by jkoritzinsky** [comment]
Fixed by a009ac0f0dc42ae0eb5752a7d602b784eb563bc5

**Comment by devsko** [comment]
Thanks
